src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+  }
+
+  static addAliveAnimals(item) {
+    Animal.alive.push(item);
+  }
```

Comment:
Consider renaming `addAliveAnimals` to a more appropriate name like `addAliveAnimal` since it adds a single animal at a time.